components/SidebarItem.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+const React = require('react');
+const classNames = require('classnames');
+
+class SidebarItem extends React.Component {
+  static getTitle (file) {
```

Comment:
While I like the use of a static method here, most students won't know what this is. I think we can extract this logic out into a function completely separate from the React component? (We can still keep it in the same file)